CHANGELOG.md

```diff
@@ -34,6 +34,9 @@
 - Dev: Add HTTP Caching headers. (#417)
 - Dev: Add custom middleware to strip trailing slashes. (#422)
 - Dev: Make cache timeout durations configurable. (#419)
+- Dev: Add prometheus middleware for chi. (#420)
+- Dev: Only create prometheus metrics for router patterns. (#425)
```

Comment:
Make only one changelog entry, treat this PR as if it's the first that handles router prometheus metrics